apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-notes-panel.presenter.tsx

```diff
@@ -10,17 +12,52 @@ interface SelfNotesPanelProps {
 export const SelfNotesPanelPresenter = ({
     compositeIdentifier: { type: resourceType, id },
     notes,
-}: SelfNotesPanelProps): JSX.Element => (
-    // TODO remove magic string
-    <div data-testid={'selfNotesPanel'}>
-        <h2>
-            {/* TODO Use standard formatter */}
-            Notes for {resourceType}/{id}
-        </h2>
-        <div>
-            {notes.length > 0
-                ? notes.map((note) => <SelfNotePresenter {...note} key={note.id} />)
-                : 'No Notes Found'}
-        </div>
-    </div>
-);
+}: SelfNotesPanelProps): JSX.Element => {
+    /**
+     * Temporary logic for drawer
+     */
+
+    const [selfNotesPanelState, setSelfNotesPanelState] = useState(false);
+
+    const handleDrawerToggle = () => {
+        setSelfNotesPanelState(!selfNotesPanelState);
+    };
+
+    const drawerWidth = '600px';
+
+    return (
+        // TODO remove magic string
+        <>
+            <Box sx={{ mt: 2.5 }}>
+                <Button variant="contained" onClick={handleDrawerToggle}>
+                    Notes
+                </Button>
+            </Box>
+
+            <Drawer
+                variant="temporary"
+                anchor="right"
+                open={selfNotesPanelState}
+                onClose={handleDrawerToggle}
+                sx={{
+                    width: drawerWidth,
+                    [`& .MuiDrawer-paper`]: { width: drawerWidth, boxSizing: 'border-box' },
+                }}
+            >
+                <div data-testid={'selfNotesPanel'}>
+                    <Box sx={{ padding: 2 }}>
+                        <h2>
+                            {/* TODO Use standard formatter */}
+                            Notes for {resourceType}/{id}
+                        </h2>
+                        <div>
+                            {notes.length > 0
+                                ? notes.map((note) => <SelfNotePresenter {...note} key={note.id} />)
+                                : 'No Notes Found'}
```

Comment:
This message should be part of the config. Good candidate for localization \ translation into indigenous languages.